### PR TITLE
Dev 161 update frontend config

### PR DESCRIPTION
**Pull request by simon-hng**
Had to add Typescript because using CVA without really sucks

### CODE REVIEW

File: styles/fonts.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { Inter } from '@next/font/google'
+
+const inter = Inter()
```
Comment (simon-hng) [inline styles/fonts.ts:3]: Added Inter font because that's a font I like

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        "./components/ui/*"
+      ],
+      "@styles/*": [
+        "./styles/*"
+      ],
+      "~/*": [
+        "./src/*"
+      ]
+    },
+    "strict": false
```
Comment (simon-hng) [inline tsconfig.json:34]: Strict mode disabled so we could theoretically still use vanilla JS ;)

File: styles/fonts.ts (hunk 1 above)
Comment (MunzerDw) [reply styles/fonts.ts:3]: perfect

File: components/Layout.tsx
Comment (MunzerDw) [inline components/Layout.tsx:1]: very cool

**Review by MunzerDw — APPROVED** [review]

**Pull request merged**